Source/Blazorise.Tailwind/Providers/TailwindClassProvider.cs

```diff
@@ -2100,6 +2100,9 @@ public override string Flex( FlexDefinition flexDefinition )
         if ( flexDefinition.GrowShrink != FlexGrowShrink.Default && flexDefinition.GrowShrinkSize != FlexGrowShrinkSize.Default )
             sb.Append( breakpoint ).Append( ToGrowShrink( flexDefinition.GrowShrink ) ).Append( "-" ).Append( ToGrowShrinkSize( flexDefinition.GrowShrinkSize ) );
 
+        if ( flexDefinition.Basis && flexDefinition.BasisSize != FlexBasisSize.Default )
```

Comment:
This doesn't produce expected result when breakpoint is "requested".

```
 <Div Flex="Flex.Row">
                    <Div Flex="Flex.Basis.Auto.OnTablet" Padding="Padding.Is2" Style="@flexItemStyle">
                        Auto
                    </Div>
                    <Div Flex="Flex.Basis.Is0.OnTablet" Padding="Padding.Is2" Style="@flexItemStyle">
                        Zero
                    </Div>
                    <Div Flex="Flex.Basis.Full.OnTablet" Padding="Padding.Is2" Style="@flexItemStyle">
                        Full
                    </Div>
                </Div>
 ```

outputs classes like: `md:basis-md:full`, which is not valid (should be `md:basis-full`)

Reply:
Good catch. It's fixed now.